framework/src/main/java/org/checkerframework/common/returnsrcvr/qual/BottomThis.java

```diff
@@ -0,0 +1,16 @@
+package org.checkerframework.common.returnsrcvr.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.SubtypeOf;
+import org.checkerframework.framework.qual.TargetLocations;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+/** The bottom type for the Returns Receiver Checker's type system. */
```

Comment:
Each annotation's Javadoc should link to its manual chapter, to enable users to learn more.
The bottom type should additionally link to https://checkerframework.org/manual/#bottom-type.
See how other qualifiers do this, for example https://checkerframework.org/api/org/checkerframework/checker/nullness/qual/KeyForBottom.html .

Even more importantly, say what the annotation means.  (The fact that it's the bottom type is relevant but isn't the most important information.)  For example, is `null` the only value of this type?

These comments apply to other annotations as well.

Reply:
Hi @mernst, I see this documentation in the manual for other bottom types:

> This is the bottom type for the XYZ type system. It should never need to be written by the programmer.

Is that sufficient here?  I'm honestly not even sure what values (if any) take on the bottom type in the returns receiver type system.

Reply:
Yes, that is often sufficient.  Usually the only value of the bottom type is `null`, but sometimes that is different.  Knowing what values helps readers to understand the type.  If you can't figure it out, you could leave that off.